Directory.Packages.props

```diff
@@ -10,7 +10,7 @@
     <PackageVersion Include="OpenTelemetry.API" Version="1.7.0" />
 
     <!-- Compatibility -->
-    <PackageVersion Include="System.Text.Json" Version="$(SystemVersion)" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.4" />
```

Comment:
Maybe add a comment to bring back `SystemVersion` after we upgrade to the latest .NET?

Reply:
Actually `SystemVersion` seems to block dependabot and `dotnet updated`, maybe we shouldn't use it...?

Reply:
Huh. Maybe? No idea what's going to happen after .NET 9 GA though. Is it going to try and update npgsql 8 dependencies to .NET9?

Reply:
I think so... We'll see :)